client/src/boot/apollo/getGraphqlFragments.ts

```diff
@@ -30,12 +30,11 @@ const getGraphqlFragments = async (baseUrl, preferStatic = true) => {
   }
   const [primaryURL, fallbackURL] = urls;
 
-  const fetchConfig = {
+  const fetchConfig: RequestInit = {
     method: 'GET',
     headers: {
       'Content-Type': 'application/json'
     },
-    uri: `${baseUrl}`,
```

Comment:
This is a good example of a potential bug that TS can highlight. In this case `uri` is a useless value. The `fetch` method will just ignore it, but TS will throw an error. You could update `uri` and scratch your head to understand why you are not getting the write info. TS saves your from this potential problem.